questions/questions/questions.py

```diff
@@ -28,12 +28,13 @@ def _generate_response(self):
             'answer_options': self.answer_options,
             'question_params': self.question_params,
             'key': self.key,
-            'answer': self.answer
+            'answer': self.answer,
+            'help_steps': self.help_steps
         }
 
     def _add_to_cache(self):
         key = self.response['key']
-        data = {**self.response, **{'answer': self.answer}}
+        data = {**self.response, **{'answer': self.answer}, **{'help_steps': self.help_steps}}
```

Comment:
![image](https://user-images.githubusercontent.com/13226802/96869177-228a9480-143d-11eb-93f5-4e0931d4dd38.png)

help_steps is already in self.response.  Instead of adding it as a separate key here, pull it out of the response in the view.